resources/dicts/patrols/forest/border/any.json

```diff
@@ -1725,7 +1725,7 @@
         },
         "weight": 20,
         "intro_text": "While on patrol, app1 notices some suspicious pawprints in the mud beneath the canopy.",
-        "decline_text": "{PRONOUN/app1/subject/CAP} {VERB/app1/decide/decides} not to investigate.",
+        "decline_text": "They decide not to investigate.",
```

Comment:
Why change the pronoun taggings on this one?

Since in the intro text it was specifically App one.

Could be 'App1 decides not to investigate, but I don't think having the 'They' here works, even if the two apps, because of how the intro text reads, if that makes sense. Just my opinion, I figured I'd skim through it to see if there's anything obvious.